salesagility/SuiteCRM
# Fixed - Missing icons for custom modules

## Description



This change adds a new icon to newly created modules in module builder. So that have a SVG icons. It also has a fall back for old custom modules so that hey at least get the basic icon for the dashlet header, sub panel header, and the recent links in the side bar. This change doesn't create icons for the actions in the side bar.

This change does not effect the previous gif icons.

## Motivation and Context

This should reduce the complaints for missing icons. As users will get the basic icon as a placeholder for missing icons.

## How To Test This

Test 1:

1. Create a new module for each template type
2. Access records of that module
3. You should see the new template icons in the sidebar, dashlet, etc.

Test 2:

1. Test old custom modules to see if the basic cube appears as described in the description
2. You should see the basic cube

Test 3:

1. Check the other themes to ensure that they get the gif instead of the SVG's.

## Types of changes


### Final checklist
